t/lib/server.lua

```diff
@@ -22,6 +23,13 @@ function _M.limit_conn()
     ngx.say("hello world")
 end
 
+function _M.plugin_proxy_rewrite()
+    local req_info = {}
+    req_info['uri'] = ngx.var.uri
+    req_info['host'] = ngx.var.host
+    req_info['scheme'] = ngx.var.scheme
+    ngx.say(json_encode(req_info))
```

Comment:
The result of json serialization is unstable, such as the order of the different keys.

So it's safer to print the value of each item directly.

```lua
ngx.say("uri: ", ngx.var.uri)
ngx.say("host: ", ngx.var.host)
...
```